fields/html/field.php

```diff
@@ -21,7 +21,7 @@
 
 }else{
 	$html_template = $field[ 'config' ][ 'default' ];
-	$content = do_shortcode( Caldera_Forms::do_magic_tags(  $html_template   ) );
+	$content = do_shortcode( Caldera_Forms::do_magic_tags( wpautop( $html_template ) ) );
```

Comment:
@iamntz Not a blocker, but it concerns me as your  commit messages become part of our git history, that your commit message doesn't describe the reason for the commit, which happens here.

A better commit message would be something like
Applies wpautop formatting to html field front-end display #2619